src/Entity/Talk.php

```diff
@@ -46,9 +46,15 @@ class Talk
      */
     private $submits;
 
+    /**
+     * @ORM\OneToMany(targetEntity=User::class, mappedBy="talks")
+     */
+    private $authors;
```

Comment:
Why did you add this new property? more over this seems wrong (never updated, never used)

Reply:
Isn't a relation needed between Talk and User to allow the speakers to be displayed on the talks list view ?